### PR TITLE
Doc: Install instructions for Inkscape 1.0

### PR DESCRIPTION
**Pull request by jcwinkler**
I would like to provide instructions on how to test current development version of TexText with Inkscape 1.0 beta while Inkscape 0.94.x is also installed and on the system path.

This PR offers instructions for Linux, as I use it. You can see it here: 

https://jcwinkler.github.io/textext/install/linux-beta.html

Windows will follow.

If some more convenient ways are possible please let me know. Thanks!

### CODE REVIEW

File: docs/source/install/linux-beta.rst [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+        chmod u+x Inkscape-fe3e306-x86_64.AppImage
+
+3. Extract the AppImage:
+
+    .. code-block:: bash
+
+        ./Inkscape-fe3e306-x86_64.AppImage --appimage-extract
+
+After this operation |Inkscape| will reside in ``~/squashfs-root/bin``. You can test it by
```
Comment (sizmailov) [inline docs/source/install/linux-beta.rst]: Doesn't it extract to current folder? Do you assume here that CWD in $HOME?

Comment (jcwinkler) [reply docs/source/install/linux-beta.rst]: Good catch. It is extracted into the current directory. Thanks! I update the doc.

**Comment by jcwinkler** [comment]
Here the Windows version, basically the same procedure as on Linux:

https://jcwinkler.github.io/textext/install/windows-beta.html

**Comment by jcwinkler** [comment]
> I would merge this PR to master in order to deploy docs automatically.

@sizmailov I will do some further cleanup in other parts of the code. After additional PR I will merge all this stuff into master while also providing a pre-release.

**Pull request merged**